### DISCUSSION TITLE
Market mechanics

### DISCUSSION THREAD

**Discussion by RainVaporeon** [DiscordDungeons/Bugs#966] [Ideas]
Make it so there could be a request system e.g. upping the market value of something or decreasing it (the cap) so more people is willing to buy/sell them

**Comment by Feathers8** [comment]
You do know about that google form, right?

**Reply by RainVaporeon** [reply]
No

**Reply by Feathers8** [reply]
Well then here https://forms.gle/7wCEdtkoJbMWHNyP9
